joern-cli/frontends/pysrc2cpg/src/main/scala/io/joern/pysrc2cpg/PythonAstVisitor.scala

```diff
@@ -662,7 +662,8 @@ class PythonAstVisitor(
           "",
           lineAndColumn,
           arguments,
-          keywordArguments
+          keywordArguments,
+          None
```

Comment:
These `None` values are already implicitly added in the function definition `methodFullName: Option[String] = None` so you can revert these explicit `None`s

Reply:
Done